### PR TITLE
CB-4269 - Propogate assumer identity property to IDBroker config

### PR DESCRIPTION
**Pull request by pdarvasi**
https://jira.cloudera.com/browse/OPSAPS-53929?focusedCommentId=2578476&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2578476

### CODE REVIEW

**Comment by pdarvasi** [comment]
I am waiting for Sandeep More's fix to be backported, but the build failed:
http://release.infra.cloudera.com/#/two_pass?stack=CDH&release=7.0.2.2-13&osname=centos7

File: template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/CMRepositoryVersionUtil.java [hunk 1]
```diff
@@ -34,15 +34,20 @@ public static boolean isEnableKerberosSupportedViaBlueprint(ClouderaManagerRepo
...
 
     public static boolean isKeepHostTemplateSupportedViaBlueprint(ClouderaManagerRepo clouderaManagerRepoDetails) {
-        LOGGER.info("ClouderaManagerRepos compared for host template");
+        LOGGER.info("ClouderaManagerRepo is compared for host template");
         return isVersionNewerOrEqualThanLimited(clouderaManagerRepoDetails::getVersion, CLOUDERAMANAGER_VERSION_6_3_0);
     }
 
     public static boolean isKnoxGatewaySupported(ClouderaManagerRepo clouderaManagerRepoDetails) {
-        LOGGER.info("ClouderaManagerRepos compared for knox gateway support");
+        LOGGER.info("ClouderaManagerRepo is compared for knox gateway support");
```
Comment (lajosrodek) [inline template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/CMRepositoryVersionUtil.java]: Minor: Use casing `Knox Gateway`.

File: template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/CMRepositoryVersionUtil.java [hunk 2]
```diff
@@ -34,15 +34,20 @@ public static boolean isEnableKerberosSupportedViaBlueprint(ClouderaManagerRepo
...
         return isVersionNewerOrEqualThanLimited(clouderaManagerRepoDetails::getVersion, CLOUDERAMANAGER_VERSION_6_4_0);
     }
 
+    public static boolean isIdBrokerManagedIdentitySupported(ClouderaManagerRepo clouderaManagerRepoDetails) {
+        LOGGER.info("ClouderaManagerRepo is compared for id broker managed identity support");
```
Comment (lajosrodek) [inline template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/CMRepositoryVersionUtil.java]: Minor: Use casing `ID Broker`.

**Comment by pdarvasi** [comment]
Thanks for the comments, @lajosrodek could you pls. merge this as it is quite urgent and I will open a follow-up with the fixes in the next minutes.

**Pull request merged**